content/en/blog/_posts/2022-08-15-introducing-cosi.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: 'Introducing COSI: Object Storage Support in Kubernetes'
+date: 2022-08-15
+slug: cosi-object-storage-support
+---
+
+**Authors:** Sidhartha Mani (alcion.ai)
+
+This article introduces the Container Object Storage Interface (COSI), a standard for provisioning and consuming object storage in Kubernetes. It is a alpha feature in Kubernetes v1.25.
+
+## Motivation
+
+File and block storage are treated as first class citizens in the Kubernetes ecosystem via CSI. Workloads using CSI volumes enjoy the benefits of portability across vendors and across Kubernetes clusters without the need to change application manifests. An equivalent standard does not exist for Object storage.
+
+Object storage has been rising in popularity in recent years as an alternative form of storage to filesystems and block devices. Object storage paradigm promotes disaggregation of compute and storage. This is done by making data available over the network, rather than locally. Disaggregated architectures allow compute workloads to be stateless, which consequently makes them easier to manage, scale and automate.
```

Comment:
```suggestion
Object storage has been rising in popularity in recent years as an alternative form of storage to filesystems and block devices. The object storage paradigm promotes disaggregation of compute and storage. This is done by making data available over the network, rather than locally. Disaggregated architectures allow compute workloads to be stateless, which consequently makes them easier to manage, scale and automate.
```

Reply:
Consider hyperlinking to (eg) the Wikipedia entry on object storage.